tests/privateZones.test.ts

```diff
@@ -1,7 +1,7 @@
 import './__mocks__/MediaStream.mock';
 import './__mocks__/navigator.mock';
-import createSpatialAudio from '../src/spatialAudio';
-import { SpatialAudio, Zone } from '../src/types/SpatialAudio';
+import createPrivateZones from '../src/privateZones';
+import { PrivateZones, Zone } from '../src/types/PrivateZones';
 
 var spatialEnvironment: any;
```

Comment:
we could switch to let. 
It should not be any type. Is it not the same as the one defined in the sdk ? 
We could define proper type is not or at least use unknown instead of any.